dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/ValueType.java

```diff
@@ -191,8 +191,8 @@ public boolean isAggregatable( AggregationType aggregationType )
         {
             return false;
         }
-
-        if ( this == TEXT || this == LONG_TEXT || this == LETTER )
+        // ?Only TEXT can have AggregationType.NONE
```

Comment:
I think that there is an additional "?"

Reply:
:-) I just said I am not sure, but I will remove it

Reply:
done